packages/expo-linear-gradient/src/NativeLinearGradient.web.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { LayoutRectangle, View } from 'react-native';
-import normalizeColor from 'react-native-web/src/modules/normalizeColor';
+import normalizeColor from 'react-native-web/dist/modules/normalizeColor';
```

Comment:
Maybe better to investigate adding web support to https://github.com/facebook/react-native/tree/master/packages/normalize-color -- [`@react-native/normalize-color`](https://www.npmjs.com/package/@react-native/normalize-color)